lib/Notification/Notifier.php

```diff
@@ -100,6 +105,11 @@ public function prepare(INotification $notification, string $languageCode): INot
 				if (!$boardId) {
 					throw new AlreadyProcessedException();
 				}
+
+				$card = $this->cardMapper->find($cardId);
+				$stackId = $card->getStackId();
+				$stack = $this->stackMapper->find($stackId);
```

Comment:
Is there a shorter way for this?

Reply:
Not as of right now as we always use an entity and don't have any way to map joined results to it currently. I'd say we keep it like this and if we encounter more places where this would be useful we can add a helper to do this in one query.